src/eos-metrics-instrumentation.c

```diff
@@ -351,6 +338,22 @@ record_stop_for_login (GQuark   session_id_quark,
                                           NULL /* auxiliary_payload */);
 }
 
+static void
+add_session (const char *session_id,
+             const char *session_path,
+             guint32 user_id)
```

Comment:
nit: `user_id` should be vertically aligned with `session_id` and `session_path`.